File: 2-semestre/uri-online-judge/java/1019.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        int segEvento, horas, minutos, segundos;
        
        Scanner entrada = new Scanner(System.in);
        
        segEvento = entrada.nextInt();
        
        horas = ((segEvento / 60) / 60) % 60;
        minutos = (segEvento / 60) % 60;
        segundos = segEvento % 60;
        
        System.out.printf("%d:%d:%d\n", horas, minutos, segundos);
    }
}

File: 2-semestre/uri-online-judge/c/1019.c
#include <stdio.h>

int main() {
    int segEvento, horas, minutos, segundos;
    
    scanf("%d", &segEvento);
    horas = ((segEvento / 60) / 60) % 60;
    minutos = (segEvento / 60) % 60;
    segundos = segEvento % 60;
    
    printf("%d:%d:%d\n", horas, minutos, segundos);
    
    return 0;
}
